app-i18n/fcitx-gtk/fcitx-gtk-5.1.0.ebuild

```
@@ -0,0 +1,56 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake gnome2-utils xdg
+
+MY_PN="fcitx5-gtk"
+S="${WORKDIR}/${MY_PN}-${PV}"
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_PN}-${PV}.tar.xz -> ${P}.tar.xz"
+KEYWORDS="~amd64 ~arm64 ~loong ~riscv ~x86"
```

```suggestion
KEYWORDS="~amd64 ~x86"
```
We should follow the standard [keywording procedure for new pacakges](https://devmanual.gentoo.org/keywording/index.html#keywording-new-packages); amd64 and x86 are fine, but other arches should be tested by the arch teams, you can file a keywording request after this is merged.

---

I wouldn't trust x86 automatically, and even so this has been tested by @liangyongxiang on amd64, arm64, loong and riscv specifically.

https://github.com/gentoo/gentoo/pull/34066#discussion_r1412210031

---

For x86, I tested it in a libvirt VM.